app/javascript/components/key-performance-indicators/date-range-dialog/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  FormControl,
+  DialogActions,
+  Button
+} from "@material-ui/core";
+import {
+  KeyboardDatePicker,
+  MuiPickersUtilsProvider
+} from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
+
+import { useI18n } from "../../i18n";
+
+const DateRangeDialog = ({ open, onClose, currentRange, setRange }) => {
+  const i18n = useI18n();
+  const [from, setFrom] = useState(currentRange.from);
+  const [to, setTo] = useState(currentRange.to);
+
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <DialogTitle>
+        {i18n.t("key_performance_indicators.date_range_dialog.title")}
+      </DialogTitle>
+      <DialogContent>
+        <DialogContentText>
+          {i18n.t("key_performance_indicators.date_range_dialog.description")}
+        </DialogContentText>
+        <FormControl>
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <KeyboardDatePicker
+              variant="inline"
+              // NOTE: using translations here causes an error, probably becuase of missing
+              //       translations. How do we handle translations in tests?
+              format="dd/MM/yyyy" // {i18n.t('key_performance_indicators.long_date_format')}
```

Comment:
Make `dd/MM/yyyy` a constant in config/constants.js incase we need to change in future

Reply:
Actually I'll make a pr to add. And you can pull later today and use.

Reply:
You'll need to change here and a few other places that you specify a date format.